### PR TITLE
Update cookbook sortable

### LINKED ISSUE #4326: Update Cookbook recipe for Sortable behaviour
### Environment

* PHP 7.0.14
* Sonata-Admin 3.13.0
* pixassociates/sortable-behavior-bundle 1.2

## Subject

Using Sonata/Admin with Sortable-Behaviour as described in [Cookbook "Sortable behaviour" ](https://sonata-project.org/bundles/admin/3-x/doc/cookbook/recipe_sortable_listing.html) will lead to an exception, since pixassociates/sortable-behavior-bundle v1.1.

This is due of the fact the author of this package silently removed the "$last_position" handling, which is not needed anymore. See commit https://github.com/pix-digital/pixSortableBehaviorBundle/commit/5c3d1a122f8d177e817c19e2fe637bbe9847bde8

This was already discussed https://github.com/sonata-project/SonataAdminBundle/issues/4299#issuecomment-276642050 and https://github.com/pix-digital/pixSortableBehaviorBundle/issues/57.

As I'm not familiar in how to upgrade the docs, I'll ask you for help.
Should I make a PR with "since Version v1.1 it's no longer necessary to handle "$last_position" [...]" or remove the paragraph entirely?

## Steps to reproduce

* Install latest sonata-project/admin-bundle
* Install pixassociates/sortable-behavior-bundle at least v1.1
* Follow [Cookbook "Sortable behaviour" ](https://sonata-project.org/bundles/admin/3-x/doc/cookbook/recipe_sortable_listing.html)
* Open the Admin's list view

## Expected results

A list view to work with.

## Actual results

An exception.

### PR DESCRIPTION
**Pull request by mvhirsch**
I am targetting this branch, because the documentation needs to be updated.

Closes #4326 

## Changelog

Updated cookbook recipe for SortableBehaviour.

```markdown

### Fixed
- Fixed cookbook recipe for SortableBehaviour.
```

## Subject

It updates the cookbook recipe when using pixassociates/sortable-behavior-bundle 1.1 or higher.
Please also see #4326

### CODE REVIEW

File: Resources/doc/cookbook/recipe_sortable_listing.rst [hunk 1]
```diff
@@ -28,7 +28,7 @@ Configuration
 Bundles
 ^^^^^^^
 - install ``gedmo/doctrine-extensions`` bundle in your project (check ``stof/doctrine-extensions-bundle`` for easier integration in your project) and enable the sortable feature in your config
-- install ``pixassociates/sortable-behavior-bundle`` in your project
+- install ``pixassociates/sortable-behavior-bundle`` at least version 1.1 in your project
```
Comment (greg0ire) [inline Resources/doc/cookbook/recipe_sortable_listing.rst]: You can use `^1.1`, people will get it and copy it in their composer.json.

**Comment by greg0ire** [comment]
> On my phone. Are red things tabs?

They are, but it's in removed code, so it's good :)

File: Resources/doc/cookbook/recipe_sortable_listing.rst (hunk 1 above)
Comment (jordisala1991) [inline Resources/doc/cookbook/recipe_sortable_listing.rst]: ^1.2 is better

^1.1 have wrong icons by default.

Comment (greg0ire) [inline Resources/doc/cookbook/recipe_sortable_listing.rst]: ^1.2 is at least 1.1 , we're just saying what is necessary for this guide to be valid.

Comment (mvhirsch) [inline Resources/doc/cookbook/recipe_sortable_listing.rst]: > You can use ^1.1, people will get it and copy it in their composer.json.

@greg0ire What about changing these lines into following?

Install sortable-behavior-bundle: ```composer require pixassociates/sortable-behavior-bundle ^1.1```

Same goes for gedmo/doctrine-extensions.

Comment (greg0ire) [inline Resources/doc/cookbook/recipe_sortable_listing.rst]: I thought about it but then what if version 2 is out? recommending `composer require pixassociates/sortable-behavior-bundle` with no argument is best IMO.

Comment (mvhirsch) [inline Resources/doc/cookbook/recipe_sortable_listing.rst]: OK :-)

**Comment by greg0ire** [comment]
Thanks @mvhirsch !

**Pull request merged**